optaplanner-core/src/main/java/org/optaplanner/core/api/domain/entity/PlanningEntity.java

```diff
@@ -68,25 +69,28 @@
      * Process B (1GB RAM), Process A (2GB RAM), Process C (7GB RAM),
      * <p>
      * Do not use together with {@link #difficultyWeightFactoryClass()}.
+     *
      * @return {@link NullDifficultyComparator} when it is null (workaround for annotation limitation)
      * @see #difficultyWeightFactoryClass()
      */
     Class<? extends Comparator> difficultyComparatorClass() default NullDifficultyComparator.class;
 
     /** Workaround for annotation limitation in {@link #difficultyComparatorClass()}. */
-    interface NullDifficultyComparator extends Comparator {}
+    interface NullDifficultyComparator extends Comparator {
+    }
 
     /**
      * The {@link SelectionSorterWeightFactory} alternative for {@link #difficultyComparatorClass()}.
      * <p>
      * Do not use together with {@link #difficultyComparatorClass()}.
+     *
      * @return {@link NullDifficultyWeightFactory} when it is null (workaround for annotation limitation)
      * @see #difficultyComparatorClass()
      */
-    Class<? extends SelectionSorterWeightFactory> difficultyWeightFactoryClass()
-            default NullDifficultyWeightFactory.class;
+    Class<? extends SelectionSorterWeightFactory> difficultyWeightFactoryClass() default NullDifficultyWeightFactory.class;
```

Comment:
@rsynek Another example of removing "manual" line break of a different type. It's impossible to insert a line break anywhere on this line other than inside the parentheses around method parameters.